packages/node/src/app/settings.ts

```diff
@@ -1,5 +1,6 @@
 import { ValidationError } from '@segment/analytics-core'
 import { HTTPClient, HTTPFetchFn } from '../lib/http-client'
+import { OAuthSettings } from '../lib/token-manager'
```

Comment:
Can we move the OAuthSettings interface to this file and create a derived type, per our convo below?